astropy/io/fits/tests/test_fitstime.py

```diff
@@ -95,7 +95,7 @@ def test_time_to_fits_loc(self, table_types):
 
         with pytest.raises(ValueError) as err:
             table, hdr = time_to_fits(t)
-            assert "Multiple Time Columns with different geocentric" in str(err.value)
+        assert "Multiple Time Columns with different geocentric" in str(err.value)
```

Comment:
The proper fix would be to use `pytest.raises(ValueError, match=...)` instead.

Reply:
Sounds good, will make this change